### PR TITLE
898 u bus

### PR DESCRIPTION
**Pull request by adriguir**

### CODE REVIEW

File: dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp [hunk 1]
```diff
@@ -56,6 +56,13 @@ ModelBusContainer::add(const shared_ptr<ModelBus>& model) {
   models_.push_back(model);
 }
 
+void
+ModelBusContainer::resetCurrentUStatus() {
+  vector<shared_ptr<ModelBus> >::const_iterator itB;
```
[the remark this answers is not in the document]
Comment (adriguir) [reply dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp]: Solved

**Comment by rosiereflo** [comment]
Could you please also solve the last 4 sonar code smells? thanks!

**Comment by adriguir** [comment]
I have corrected the last four code smells (I didn't do it at first glance yesterday because you self-assigned yourself so I thought you was doing it in another dev) but whatever, it's done now.

I don't know what I wrongly do in the process but I still have conflicts, whereas I rebase on the master before pushing. Would you have a suggestion other than closing this PR and open a new one with a branch based on the more recent version of the master?

**Comment by rosiereflo** [comment]
The assignment to me was automatic :)
No I don't have any other option to propose. Let's open a new PR that I will accept directly.

**Comment by adriguir** [comment]
> The assignment to me was automatic :)
> No I don't have any other option to propose. Let's open a new PR that I will accept directly.

No problem but quite strange behavior, because only four of them were automatically assigned. Do you know what is the reason?

I will try a last time to resolve locally and push and will see what happens, otherwise I will open a new PR.

**Comment by rosiereflo** [comment]
> No problem but quite strange behavior, because only four of them were automatically assigned. Do you know what is the reason?

Probably that sonar identified me as the author of this part of the code.

**Comment by adriguir** [comment]
> > No problem but quite strange behavior, because only four of them were automatically assigned. Do you know what is the reason?
> 
> Probably that sonar identified me as the author of this part of the code.

Sounds a good explanation. I think that then only Gautier and you will have automatic assignments as you are the only ones with proper Sonar accounts.

**Pull request merged**